Assets/Scripts/MovementPC.cs

```diff
@@ -5,43 +5,50 @@
 
 public class MovementPC : MonoBehaviour
 {
-    public float playerPositionX;
-    public float playerPositionY;
+    private float playerPositionX;
+    private float playerPositionY;
+    
+    public float maxPlayerX;
+    public float maxPlayerY;
+
+    public float playerSpeedX;
+    public float playerSpeedY;
```

Comment:
```suggestion
    private float _playerPositionX;
    private float _playerPositionY;
    
    [SerializeField] private float maxPlayerX;
    [SerializeField] private float maxPlayerY;

    [SerializeField] private float playerSpeedX;
    [SerializeField] private float playerSpeedY;
```

Meer protection en handiger.